pkg/resolver/related_readme.md

```diff
@@ -0,0 +1,63 @@
+**Search Term** = Resource(s) searched for from the search bar. 
```

Comment:
@jlpadilla Added a small readme here. We can move it somewhere else, if it doesn't fit here, or remove it altogether, if it is covered elsewhere.

Reply:
Thanks!  Maybe we can add a link from the v2-operator wiki after we merge.
https://github.com/stolostron/search-v2-operator/wiki